pageserver/src/page_service.rs

```diff
@@ -1636,13 +1643,17 @@ impl PageServerHandler {
                             return Err(e);
                         }
                     };
-                    self.pagesteam_handle_batched_message(
-                        pgb_writer,
-                        batch,
-                        io_concurrency.clone(),
-                        &cancel,
-                        protocol_version,
-                        &ctx,
+                    warn_slow(
+                        "getpage request batch",
+                        WARN_SLOW_GETPAGE_THRESHOLD,
+                        self.pagesteam_handle_batched_message(
```

Comment:
Note that this batch will contain either one or more get page requests or one of the other request types (e.g. dbsize). Might be good to peek into the batch and label the log line accordingly.

Reply:
Ah, missed that we don't do mixed batches, so this was trivial.